bolt/include/bolt/Profile/DataAggregator.h

```diff
@@ -370,33 +370,46 @@ class DataAggregator : public DataReader {
   /// memory.
   ///
   /// File format syntax:
-  /// {B|F|f|T} [<start_id>:]<start_offset> [<end_id>:]<end_offset> [<ft_end>]
-  ///       <count> [<mispred_count>]
+  /// E <event>
```

Comment:
Can you specify multiple events per file?

Reply:
Yes. It is possible to group samples by event name, but currently we don't have any logic based on the event names other than passing them through to fdata and YAML.